index.ts

```diff
@@ -285,33 +286,45 @@ const logTimeEntryToJira = async (
     LocalTime.NOON,
     ZoneId.of(userTz)
   );
-  const requestBody: JiraWorkLog = {
+  let requestBody: JiraWorkLog = {
     timeSpentSeconds: rounded_hours * 60 * 60,
     notifyUsers: false,
     comment: {
       version: 1,
       type: "doc",
+      content: [],
+    },
+    started: startedDate.format(JIRA_DATE_TIME_FORMATTER),
+  };
+
+  if (jiraMessage) {
+    requestBody.comment?.content?.push({
+      type: "paragraph",
       content: [
         {
-          type: "paragraph",
-          content: [
-            {
-              type: "emoji",
-              attrs: {
-                shortName: ":timer:",
-                id: "23f2",
-              },
-            },
-            {
-              type: "text",
-              text: `Harvest time entry ID: ${harvestId}`,
-            },
-          ],
+          type: "text",
+          text: jiraMessage,
         },
       ],
-    },
-    started: startedDate.format(JIRA_DATE_TIME_FORMATTER),
-  };
+    });
+  } else {
+    requestBody.comment?.content?.push({
+      type: "paragraph",
+      content: [
+        {
+          type: "emoji",
+          attrs: {
+            shortName: ":timer:",
+            id: "23f2",
+          },
+        },
+        {
+          type: "text",
+          text: `Harvest time entry ID: ${harvestId}`,
```

Comment:
ok so little did you know that you've now enmeshed yourself in one of the jankiest pieces of code known to man.
the way that this tool avoid re-logging harvest time entries is by sticking the entry ID in the jira worklog comment, so I think we'll want to take care to include both `jiraMessage` and this bit